### PR TITLE
feb meetup whatup

### PR DESCRIPTION
**Pull request by aaronj1335**
- [ ] @lawnsea 
- [ ] @beaktor
- [ ] @astashov

### CODE REVIEW

**Comment by astashov** [comment]
👍

**Comment by beaktor** [comment]
:+1:

File: _posts/2017-02-13-meetup.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  _posts/YYYY-MM-DD-meetup.md`, remove this README, and add a `when` property to this
+  section. the front page will update to let folks know that we've got
+  something in the works.
+title: Anton Astashov on immutability
+author: aaron
+layout: post
+categories:
+- posts
+- meetups
+when: 2017-02-21T19:30:00-05:00
```
Comment (lawnsea) [inline _posts/2017-02-13-meetup.md]: What are the odds this is correct?

File: _posts/2017-02-13-meetup.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  </div>
+</div>
+
+[Mozilla][] is providing food and drink, so go [download Firefox][] if it's not
+already your main browser and spend some time celebrating the open web.
+
+{% include give-em-the-business.html location='spredfast' %}
+
+[Mozilla]: https://www.mozilla.org
+[download Firefox]: https://www.mozilla.org/en-US/firefox/new/
```
Comment (lawnsea) [inline _posts/2017-02-13-meetup.md]: Wrong URL. Here's the correct one: https://netscape-browser.en.softonic.com/

File: _posts/2017-02-13-meetup.md (hunk 1 above)
Comment (aaronj1335) [inline _posts/2017-02-13-meetup.md]: heh, fixed to -06

File: _posts/2017-02-13-meetup.md [hunk 3]
```diff
@@ -1,16 +1,11 @@
...
-  section. the front page will update to let folks know that we've got
-  something in the works.
 title: Anton Astashov on immutability
 author: aaron
 layout: post
 categories:
 - posts
 - meetups
-when: 2017-02-21T19:30:00-05:00
+when: 2017-02-21T19:30:00-06:00
```
Comment (lawnsea) [inline _posts/2017-02-13-meetup.md]: lmao

File: _posts/2017-02-13-meetup.md (hunk 2 above)
Comment (aaronj1335) [inline _posts/2017-02-13-meetup.md]: :trollface:

**Comment by lawnsea** [comment]
👍

File: _posts/2017-02-13-meetup.md (hunk 3 above)
Comment (aaronj1335) [inline _posts/2017-02-13-meetup.md]: hi i'm aaron, and i'm a *world class* software developer

**Comment by aaronj1335** [comment]
closed by 0fcf104

**Pull request closed without merging**